e2e/cypress/integration/onboarding/cloud_sysadmin_onboarding_spec.js

```diff
@@ -7,15 +7,13 @@
 // - Use element ID when selecting an element. Create one if none.
 // ***************************************************************
 
-// Group: @enterprise @onboarding @cloud_only
+// Group: @onboarding
```

Comment:
If this is now applicable to all, could you please remove "cloud_" prefix in the filename? Thanks!